src/app/modules/registration/pages/snow/avalanche-obs/avalanche-obs.page.html

```diff
@@ -18,6 +18,15 @@
       <ion-list-header class="ion-text-uppercase">
         <ion-label> {{ 'REGISTRATION.SNOW.AVALANCHE_OBS.TITLE' | translate}} </ion-label>
       </ion-list-header>
+      <ion-item [ngClass]="{'hasWarning': dateIsDifferentThanObsTime}">
+        <ion-label
+          [color]="(!avalancheObs.DtAvalancheTime && showWarning) ? 'danger' : 'medium'"
+          position="stacked"
+          class="ion-text-uppercase"
+          >{{ 'REGISTRATION.SNOW.AVALANCHE_OBS.EARLIEST_TIME' | translate }}</ion-label
+        >
+        <app-datetime-picker [(dateTime)]="avalancheObs.DtEarliestAvalancheTime" [maxDate]="avalancheObs.DtAvalancheTime"> </app-datetime-picker>
```

Comment:
Jeg finner ingen måte å nullstille feltet på, når jeg først har registrert et tidspunkt. Dette burde vi sikkert løst på en generell måte for alle datofelter.

Reply:
Jeg har laget en @Input på app-datetime-picker som gjører at den nullstilles om man trykker på "Avbryt"